rkt/trust.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var (
 	cmdTrust = &cobra.Command{
-		Use:   "trust [--prefix=PREFIX] [--insecure-allow-http] [--skip-fingerprint-review] [--root] [PUBKEY ...]",
+		Use:   "trust [--prefix=PREFIX] [--insecure-allow-http] [--skip-fingerprint-review] [--root] [PUBKEY ...] [--overwrite-trusted]",
```

Comment:
Shouldn't the `--overwrite-trusted=...`  be before the `[PUBKEY...]`?

I find boolean parameter defaulting to true confusing because just writing `--overwrite-trusted` would have no effect.

I am not sure how to make things clearer. I overheard other suggestions but they're kind of long:
- `--disable-overwrite-trusted`
- `--prevent-trusted-from-being-overwritten`

Reply:
Agreed. `--disable-overwrite-trusted` seems to do a better job of removing the confusion while also keeping the default to `true` for backwards compatibility.

Reply:
On second thoughts how does `--dont-overwrite-trusted` sound for the name of the flag?

Reply:
Another thought, how about `--skip-trusted` ? It's more concise and also is similar to the `--skip-fingerprint-review`.

Reply:
Ack, I think `--skip-trusted` is a better choice.

Reply:
What about `--skip-already-trusted`? Too long?

Reply:
`--skip-already-trusted` does feel long. I feel `already` does not appear to add any additional meaning on `--skip-trusted`.

Reply:
Should we stick with `--skip-trusted` then?